companies-manifest.js

```diff
@@ -680,6 +681,22 @@ const companies = [
       },
     },
   },
+  {
+    key: 'https://framer.com/',
+    similarWebGlobalRank: 185723,
+    logo: FramerLogo,
+    style: {},
+    projects: {
+      framer: {
+        title: 'Framer',
+        link: 'https://framer.com',
+        src: '/static/screenshots/thumbnails/framer.com.jpg',
```

Comment:
The screenshot has the cookie banner clearly visible in it. Can you take another one without that notification? :)

Reply:
I have updated the screenshot and logo. Can you please check it?

Reply:
Great Job!